tests/phpunit/tests/admin/includesScreen.php

```diff
@@ -431,6 +456,9 @@ function test_help_tabs_priority() {
 
 	/**
 	 * @ticket 25799
+	 *
+	 * @covers WP_Screen::add_option
+	 * @covers WP_Screen::get_option
```

Comment:
```suggestion
	 * @covers WP_Screen::add_option
	 * @covers WP_Screen::get_option
	 * @covers WP_Screen::remove_option
	 * @covers WP_Screen::remove_options
```

This one is testing all of the option methods. (doing too much)